_topic_map.yml

```diff
@@ -1194,64 +1189,77 @@ Topics:
+    File: custom-builds-buildah
+    Distros: openshift-enterprise,openshift-webscale,openshift-origin
+  - Name: Performing basic builds
+    File: basic-build-operations
+    Distros: openshift-enterprise,openshift-webscale,openshift-origin,openshift-dedicated,openshift-online
+  - Name: Triggering and modifying builds
+    File: triggering-builds-build-hooks
+    Distros: openshift-enterprise,openshift-webscale,openshift-origin,openshift-dedicated,openshift-online
+  - Name: Performing advanced builds
+    File: advanced-build-operations
+    Distros: openshift-enterprise,openshift-webscale,openshift-origin
+  - Name: Using Red Hat subscriptions in builds
+    File: running-entitled-builds
+    Distros: openshift-enterprise,openshift-webscale,openshift-origin
+  - Name: Securing builds by strategy
+    File: securing-builds-by-strategy
+    Distros: openshift-enterprise,openshift-webscale,openshift-origin
+  - Name: Build configuration resources
+    File: build-configuration
+    Distros: openshift-enterprise,openshift-webscale,openshift-origin
+  - Name: Troubleshooting builds
+    File: troubleshooting-builds
+    Distros: openshift-enterprise,openshift-webscale,openshift-origin
+  - Name: Setting up additional trusted certificate authorities for builds
+    File: setting-up-trusted-ca
+    Distros: openshift-enterprise,openshift-webscale,openshift-origin,openshift-dedicated
+- Name: Pipelines
+  Dir: pipelines
   Distros: openshift-enterprise,openshift-webscale,openshift-origin
-- Name: Setting up additional trusted certificate authorities for builds
-  File: setting-up-trusted-ca
-  Distros: openshift-enterprise,openshift-webscale,openshift-origin,openshift-dedicated
----
-Name: Pipelines
-Dir: pipelines
-Distros: openshift-enterprise,openshift-webscale,openshift-origin
-Topics:
-- Name: Understanding OpenShift Pipelines
-  File: understanding-openshift-pipelines
-- Name: Installing OpenShift Pipelines
-  File: installing-pipelines
-- Name: Uninstalling OpenShift Pipelines
-  File: uninstalling-pipelines
-- Name: Creating CI/CD solutions for applications using OpenShift Pipelines
-  File: creating-applications-with-cicd-pipelines
-- Name: Working with Pipelines using the Developer perspective
-  File: working-with-pipelines-using-the-developer-perspective
-- Name: OpenShift Pipelines release notes
-  File: op-release-notes
+  Topics:
+  - Name: Understanding OpenShift Pipelines
+    File: understanding-openshift-pipelines
+  - Name: Installing OpenShift Pipelines
+    File: installing-pipelines
+  - Name: Uninstalling OpenShift Pipelines
+    File: uninstalling-pipelines
+  - Name: Creating CI/CD solutions for applications using OpenShift Pipelines
+    File: creating-applications-with-cicd-pipelines
+  - Name: Working with Pipelines using the Developer perspective
+    File: working-with-pipelines-using-the-developer-perspective
+  - Name: OpenShift Pipelines release notes
+    File: op-release-notes
+#- Name: GitOps
```

Comment:
Do we want this added, commented out? Or should that happen in a different PR?

Reply:
So we want to add this, and the entire reorg was initiated due to this component. But we don't have the content in place yet. Hence commented out. Is there a concern merging this commented out?
In the content PR we will add this with changes in the title as required, is that OK? This will come in slightly later, we spoke to Vikram and have an exception for this. Basically this component has an async release cycle from that of OCP.

Reply:
Okay.